packages/grafana-ui/src/components/Monaco/types.ts

```diff
@@ -19,11 +19,10 @@ export type MonacoEditor = monacoType.editor.IStandaloneCodeEditor;
 export type MonacoOptions = MonacoOptionsWithGrafanaDefaults;
 
 export interface CodeEditorProps {
-  value: string;
+  value: string | undefined;
```

Comment:
Changing the type or API of a shared library is a big decision that will impact other teams, let's make sure we can't avoid it first.